app/visualize/analysis/stmt/stmt_traveler.py

```diff
@@ -51,5 +54,74 @@ def _internal_travel(node: ast, elem_manager: CodeElementManager):
         elif isinstance(node, ast.Expr):
             return StmtTraveler.expr_travel(node, elem_manager)
 
+        elif isinstance(node, ast.If):
+            return StmtTraveler.if_travel(node, [], [], elem_manager)
         else:
             raise TypeError(f"[StmtTraveler] {type(node)}는 잘못된 타입입니다.")
+
+    @staticmethod
+    def if_travel(
+        node: ast.If, conditions: list[ConditionObj], body_objs: list[BodyObj], elem_manager: CodeElementManager
+    ) -> IfStmtObj:
+        # parse 조건문 : if("test") or elfi("test") 부분
+        StmtTraveler._append_condition_obj(conditions, elem_manager, node)
+
+        # parse body
+        StmtTraveler._parse_if_body(node, conditions, body_objs, elem_manager)
+
+        # parse elif or else
+        StmtTraveler._parse_if_orelse(body_objs, conditions, elem_manager, node)
+
+        return IfStmtObj(conditions=tuple(conditions), body=BodyObj(body_steps=body_objs, cur_value=0))
+
+    @staticmethod
+    def _append_condition_obj(conditions, elem_manager, node: ast.If | ast.stmt):
+        if len(conditions) == 0:  # if
+            condition = IfStmt.parse_if_condition(node.test, elem_manager)
+
+        elif isinstance(node, ast.If):  # elif
+            condition = IfStmt.parse_elif_condition(node.test, elem_manager)
+
+        else:
+            raise TypeError(f"[IfTraveler] {type(node)}는 잘못된 타입입니다.")
+
+        # 조건문 parse: if("test") or elif("test") 부분
+        condition = IfStmt.parse_condition(node.test, elem_manager)
+        conditions.append(condition)
+
+    @staticmethod
+    def _append_else_condition_obj(conditions, node: ast.stmt, result: bool):
+        if isinstance(node, ast.stmt):  # else
+            condition = IfStmt.parse_else_condition(node, result)
+
+        else:
+            raise TypeError(f"[IfTraveler] {type(node)}는 잘못된 타입입니다.")
+
+        conditions.append(condition)
+
+    @staticmethod
+    def _parse_if_body(
+        node: ast.If, conditions: list[ConditionObj], body_objs: list[BodyObj], elem_manager: CodeElementManager
+    ):
+        if conditions[-1].result is True:  # 조건절의 결과 값이 True이면 해당 body 로직 추가
+            for body in node.body:
+                body_objs.append(StmtTraveler._internal_travel(body, elem_manager))
+
+    @staticmethod
+    def _parse_if_orelse(body_objs, conditions, elem_manager, node):
+        # elif 처리
+        if isinstance(node.orelse[0], ast.If):
+            StmtTraveler.if_travel(node.orelse[0], conditions, body_objs, elem_manager)
+
+        # else 처리
+        elif isinstance(node.orelse[0], ast.stmt):
+            if len(body_objs) == 0:  # if, elif문 들의 조건 값이 모두 false 일 때 else문 의 body 추가
+                StmtTraveler._append_else_condition_obj(conditions, node.orelse[0], True)
+            else:
+                StmtTraveler._append_else_condition_obj(conditions, node.orelse[0], False)
```

Comment:
```suggestion
            is_exist_body = True if len(body_objs) == 0 else False
            
            StmtTraveler._append_else_condition_obj(conditions, node.orelse[0], is_exist_body)
```
변경되는 값이 마지막 인자만 있으면 이렇게 표현하는 건 어떨까요?

Reply:
오 좋은 방법이네요! 저도 ` if len(body_objs) == 0:`여기가 이해하기 어려운 것 같아서 고치고 싶었는데 방법이 떠오르지 않아서 못고쳤거든요

아래와 같이 수정했습니다.
```
 elif isinstance(node.orelse[0], ast.stmt):
            is_exist_body = True if len(body_objs) != 0 else False

            if not is_exist_body:
                # else문의 body 추가
                StmtTraveler._append_else_condition_obj(conditions, node.orelse[0], True)
            else:
                StmtTraveler._append_else_condition_obj(conditions, node.orelse[0], False)

            for stmt_node in node.orelse:
                body_objs.append(StmtTraveler._internal_travel(stmt_node, elem_manager))
```

Reply:
https://github.com/Co-due/Code-Analysis-Server/pull/139/commits/8d2f703c0ef17b184afad6ec5519e04b44129eae 수정했습니다.